src/platform-implementation-js/style/gmail.css

```diff
@@ -474,8 +474,8 @@ steps to repro:
 2. open non-native sidebar
 3. mole overlaps sidebar
 */
-.dw .nH > .no .nH.nn:last-child[style='width: 320px;'] {
-  width: 376px !important;
+.dw .nH > .no .nH.nn:last-child[style='order: 0; width: 310px;'] {
```

Comment:
If this selector's style attribute doesn't match exactly, we'll end up back in the same situation.

Since Google Chat enabled Gmail's mole parent uses the `order` style, this may have previously been broken when https://support.google.com/mail/answer/11555490 was release Summer '22.